components/motor/gpio/motor_encoder.go

```diff
@@ -354,184 +240,162 @@ func (m *EncodedMotor) rpmMonitor() {
 			continue
 		}
 		now := time.Now().UnixNano()
-		if now == lastTime {
-			// this really only happens in testing, b/c we decrease sleep, but nice defense anyway
+		atomic.AddInt64(&m.rpmMonitorCalls, 1)
```

Comment:
I really wonder what this is giving us. Keep it for now but let's investigate further.

Reply:
I agree didn't really see a point to this but yeah I'll leave it for now